sigs/sig-cnc-charter.md

```diff
@@ -0,0 +1,60 @@
+# CNC Special Interest Group
+
+The CNC SIG is responsible for the maintenance, governance, and administration of the CNC routers, adjacent equipment, and space at denhac.
+
+The group will respond to requests from denhac members for future needs and will help create recommendations to the board that require funding in excess of the budget provided to the SIG by the denhac board of directors.
+
+## Roles and Responsible Parties
+
+The CNC SIG will be led by two Co-Leads, and a group of appointed Trainers/Maintainers. 
+
+General responsibilities for these Roles (this is not an exhaustive list):
+
+* The SIG Co-Leads will be generally responsible for SIG project management and/or delegation of these responsibilities, as well as the general governance of the group's actions at large.
+* SIG Co-Leads are responsible for the appointments of properly qualified Maintainers and Trainers, as well as providing any extended training required for both.
+* Maintainers will support the Co-Leads in matters regarding the upkeep and management of machines.
+* Trainers will be responsible for authorizing members to use the machines safely and providing assistance with machine usage when appropriate.
+* There is not a fixed number of Maintainer and Trainer positions available; they are filled based on demonstrated skill, merit, enthusiasm, and member demand for the machines.
+* When changing leadership, responsibilities should be officially transitioned to the new party on the first day of the month following an election for new leadership, or as privately discussed between the previous and incoming leadership.
```

Comment:
please remove reference to elections - could reword as: "When changing leadership, responsibilities should be officially transitioned to the new party on the first day of the month following the change in leadership, or as privately discussed between the previous and incoming leadership."
but I think removing this line entirely would be fine too